ospo-mindmap/Img/ospomindmap.html

```diff
@@ -16,13 +16,13 @@
   height: 100vh;
 }
 </style>
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.2.0/dist/style.css">
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.2.0/dist/style.min.css">
```

Comment:
I don't know why my vesion uses the `min` file, but this should be a good thing, smaller file size with full functionality

Reply:
Ok! I've also updated the index.html within the public folder with the new html

Reply:
Thanks!